### PR TITLE
Add LabelImg

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge/help-python 
Not sure why the test fails on Linux. I'm guessing it's because the CI is headless and they're launching something but not sure.
Also, not sure which PyQT to use, couldn't find anything in the docs about that so any advice here is appreciated.

**Comment by chrisburr** [comment]
See https://github.com/conda-forge/mayavi-feedstock/issues/7

**Comment by BastianZim** [comment]
@chrisburr Thanks for the suggestion but it's still failing, or did you mean something else?

**Comment by chrisburr** [comment]
You need to start the test command with `xvfb-run` so it runs the test with a virtual screen attached. I can’t remember exactly how but hopefully that’s enough to guide you else I’ll look it up tomorrow.

**Comment by BastianZim** [comment]
Thanks, will try that.

File: recipes/LabelImg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - py2app  # [osx]
+    - python
+  run:
+    - lxml
+    - pyqt
```
Comment (BastianZim) [inline recipes/LabelImg/meta.yaml:26]: Not sure which pyqt from conda-forge to use here?

**Comment by BastianZim** [comment]
@chrisburr Fixed it! Was just a case of adding the `QT_QPA_PLATFORM=offscreen` env variable.

**Comment by BastianZim** [comment]
@conda-forge/help-python Would be ready for review, thanks!

**Comment by BastianZim** [comment]
Just a gentle ping again @conda-forge/staged-recipes

**Comment by BastianZim** [comment]
Thanks @chrisburr!

**Pull request merged**